File: aztec/src/main/kotlin/org/wordpress/aztec/watchers/event/buckets/API26Bucket.java
package org.wordpress.aztec.watchers.event.buckets;

import org.wordpress.aztec.watchers.event.sequence.known.space.API26InWordSpaceInsertionEvent;

public class API26Bucket extends Bucket{

    public API26Bucket() {
        // constructor - here add all identified sequences for this bucket
        mUserOperations.add(new API26InWordSpaceInsertionEvent());
        //mUserOperations.add(new ...);
        //mUserOperations.add(new ...);
        //mUserOperations.add(new ...);
    }
}

File: aztec/src/main/kotlin/org/wordpress/aztec/watchers/event/buckets/API26Bucket.kt
package org.wordpress.aztec.watchers.event.buckets

import org.wordpress.aztec.watchers.event.sequence.known.space.API25InWordSpaceInsertionEvent
import org.wordpress.aztec.watchers.event.sequence.known.space.API26PrependNewLineOnStyledSpecialTextEvent
import org.wordpress.aztec.watchers.event.sequence.known.space.API26PrependNewLineOnStyledTextEvent

class API26Bucket : Bucket() {
    init {
        // constructor - here add all identified sequences for this bucket
        userOperations.add(API25InWordSpaceInsertionEvent())
        userOperations.add(API26PrependNewLineOnStyledTextEvent())
        userOperations.add(API26PrependNewLineOnStyledSpecialTextEvent())
        //mUserOperations.add(new ...);
        //mUserOperations.add(new ...);
        //mUserOperations.add(new ...);
    }
}
